tox.ini

```diff
@@ -114,6 +115,15 @@ extras =
 commands =
     pytest {posargs} --disable-warnings --log-cli-level=info --log-cli-format="%(asctime)s [%(levelname)8s] %(name)s:%(lineno)s %(message)s" --log-date-format="%Y-%m-%d %H:%M:%S" --cov-report=term --cov-report=xml --cov-config=.coveragerc --cov={envsitepackagesdir}/pudl test/etl_test.py test/glue_test.py test/fast_output_test.py test/interim_etl_test.py test/datastore_test.py
 
+
+# Runs unit tests under src/pudl
+[testenv:unit]
+extras =
+    test
+commands =
+    pytest {posargs} --disable-warnings --log-cli-level=info --log-cli-format="%(asctime)s [%(levelname)8s] %(name)s:%(lineno)s %(message)s" --log-date-format="%Y-%m-%d %H:%M:%S" --cov-report=term --cov-report=xml --cov-config=.coveragerc --cov={envsitepackagesdir}/pudl src/pudl
+
+
```

Comment:
Huh, now that I see this broken out, I wonder if it doesn't make sense to break out the other individual collections of tests that we want to run, so they can be run individually easily if necessary. Is there some way we can store and re-use this long-ass pytest command? This creates a single, shared coverage report, right?

Reply:
Hah. I must admit that I do not have full understanding of tox/pytest and code coverage setup to be able to answer this question. I will look into it and see if I can figure this out.

Reply:
Actually, after reading https://docs.pytest.org/en/stable/unittest.html it seems that we can simply rely on `pytest tests` to run this. Let me adapt the code to do so.